### PR TITLE
[DO NOT SUBMIT] Updating resource name config for oneof restructuring

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
This will not be submitted here - I am making this PR for demonstration purposes.

### CODE REVIEW

File: google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml [hunk 1]
```diff
@@ -27,20 +27,37 @@ license_header:
 collection_oneofs:
 - oneof_name: parent_oneof
   collection_names:
-  - location
   - project
+  - location
```
Comment (michaelbausor) [inline google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml]: Does this reordering have an important effect?

Comment (garrettjonesgoogle) [reply google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml]: It does - the sample code generation picks the first one in the list for the example type.

Comment (michaelbausor) [reply google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml]: Yes, but why do we want project instead of location?

Comment (garrettjonesgoogle) [reply google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml]: It's a shorter resource path with fewer required inputs, so it seems like it's the most "basic."

**Comment by michaelbausor** [comment]
Changes LGTM

**Comment by shakilbhuiya** [comment]
Thanks for your supporting.

**Comment by elharo** [comment]
Please close this PR

**Comment by gj11880** [comment]
How?

**Comment by chrisdunelm** [comment]
Old demo PR. Closing.

**Pull request closed without merging**